landlab/components/lateral_erosion/lateral_erosion.py

```diff
@@ -0,0 +1,631 @@
+# -*- coding: utf-8 -*-
+"""
+Grid-based simulation of lateral erosion by channels in a drainage network.
+ALangston
+
+
+"""
+
+import numpy as np
+
+from landlab import Component
+from landlab.components.flow_accum import FlowAccumulator
+from .node_finder import node_finder
+
+
+class LateralEroder(Component):
+    """
+    Laterally erode neighbor node through fluvial erosion.
+
+    Landlab component that finds a neighbor node to laterally erode and
+    calculates lateral erosion.
+
```

Comment:
I would put a link to your paper here! Its so cool!

Also, this is where I'd put statements about hard coded coefficients (I've noted some below).

Reply:
Link to paper added.